File: app/src/main/java/com/lemonade/dima/screen/MessagesView.java
package com.lemonade.dima.screen;

import com.lemonade.dima.model.Message;
import com.lemonade.dima.model.ResponseMessages;

import java.util.List;

public interface MessagesView {
    void addMessagesToEnd(List<Message> message);

    void addMessage(Message message);

    void addMessages(ResponseMessages responseMessages);
}

File: app/src/main/java/com/k4dima/chat/messages/MessagesView.kt
package com.k4dima.chat.messages

import com.k4dima.chat.app.model.Message
import com.k4dima.chat.app.model.ResponseMessages

interface MessagesView {
    fun addMessagesToEnd(messages: List<Message>)

    fun addMessage(message: Message)

    fun addMessages(responseMessages: ResponseMessages)
}